velox/exec/tests/utils/QueryAssertions.h

```diff
@@ -217,7 +217,8 @@ bool assertEqualResults(
     const std::vector<RowVectorPtr>& actual);
 
 bool assertEqualResults(
-    const MaterializedRowMultiset& expected,
+    const MaterializedRowMultiset& expectedRows,
+    const TypePtr& expectedType,
```

Comment:
RowType & expectedRowType

Reply:
Did you mean the change as `const RowTypePtr& expectedRowType`?

Reply:
That's right.

Reply:
Fixed.